sched/task/task_setup.c

```diff
@@ -83,6 +83,7 @@ static int nxtask_assign_pid(FAR struct tcb_s *tcb)
   FAR struct tcb_s **pidhash;
   pid_t next_pid;
   int   hash_ndx;
+  void * temp;
```

Comment:
```suggestion
  void *temp;
```

Reply:
Oops, sorry. I was testing it quickly and I missed it...